releases/v1.7.9.toml

```diff
@@ -0,0 +1,33 @@
+# commit to be tagged for new release
+commit = "HEAD"
+
+# project_name is used to refer to the project in the notes
+project_name = "containerd"
+
+# github_repo is the github project, only github is currently supported
+github_repo = "containerd/containerd"
+
+# match_deps is a pattern to determine which dependencies should be included
+# as part of this release. The changelog will also include changes for these
+# dependencies based on the change in the dependency's version.
+match_deps = "^github.com/(containerd/[a-zA-Z0-9-]+)$"
+
+# previous release of this project for determining changes
+previous = "v1.7.8"
+
+# pre_release is whether to include a disclaimer about being a pre-release
+pre_release = false
+
+# preface is the description of the release which precedes the author list
+# and changelog. This description could include highlights as well as any
+# description of changes. Use markdown formatting.
+preface = """\
+The ninth patch release for containerd 1.7 contains various fixes and updates.
+
+### Notable Updates
+
+* **Update hcsshim tag to v0.11.4 ([#9326](https://github.com/containerd/containerd/pull/9326))
+* **Expose usage of deprecated features ([#9315](https://github.com/containerd/containerd/pull/9315))
+* **cri: add deprecation warnings for mirrors, auths, and configs ([#9327](https://github.com/containerd/containerd/pull/9327))
```

Comment:
nit: looks like these 3 notable updates have a start bold (`**`) in markdown but don't end bold on each line?

Reply:
Hi @estesp , I was using the other toml for previous releases as an example before sending out the PR.
![image](https://github.com/containerd/containerd/assets/99994218/9b16780c-d4f0-4ca2-8050-a3d28547108a)


I'll end these lines with bold as you suggested and push a new commit

Reply:
has anything with the release tool changed? I do not see them produce the output as expected

Reply:
I don't remember having to do any of these explicitly in the previous PRs I've sent out to cut new tags